api/src/modules/order_group/order_group.controller.ts

```diff
@@ -0,0 +1,20 @@
+import { Body, Controller, Post, Req, UseGuards } from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+import { CreateOrderGroupDto } from './dto/create.dto';
+import { RequestWithUser } from 'src/types/requests.type';
```

Comment:
khai báo module path types vào tsconfig => 
 "@types/*": ["src/types/*"]
rồi sử dụng lại như sau 
```suggestion
import { RequestWithUser } from '@types/requests.type';
```